hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/services/TestApacheClientConnectionPool.java

```diff
@@ -76,14 +77,46 @@ public void testPoolWithZeroSysProp() throws Exception {
   @Test
   public void testEmptySizePool() throws Exception {
     Configuration configuration = new Configuration();
-    configuration.set(FS_AZURE_APACHE_HTTP_CLIENT_MAX_CACHE_CONNECTION_SIZE, "0");
-    AbfsConfiguration abfsConfiguration = new AbfsConfiguration(configuration, EMPTY_STRING);
-    try (KeepAliveCache keepAliveCache = new KeepAliveCache(abfsConfiguration)) {
-      Assertions.assertThat(keepAliveCache.put(Mockito.mock(HttpClientConnection.class))).isFalse();
-      Assertions.assertThat(keepAliveCache.get()).isNull();
+    configuration.set(FS_AZURE_APACHE_HTTP_CLIENT_MAX_CACHE_CONNECTION_SIZE,
+        "0");
+    AbfsConfiguration abfsConfiguration = new AbfsConfiguration(configuration,
+        EMPTY_STRING);
+    try (KeepAliveCache keepAliveCache = new KeepAliveCache(
+        abfsConfiguration)) {
+      assertCachePutFail(keepAliveCache,
+          Mockito.mock(HttpClientConnection.class));
+      assertCacheGetNull(keepAliveCache);
     }
   }
 
+  private void assertCacheGetNull(final KeepAliveCache keepAliveCache)
```

Comment:
can you use isNull and isNotNull to make clearer when reading what is being asserted?

Reply:
Makes sense. Have rename to `assertCacheGetIsNul`l and `assertCacheGetIsNonNull`